daml.yaml

```diff
@@ -4,7 +4,7 @@
 sdk-version: 2.7.0-snapshot.20230719.11983.0.vd999a21a
 name: daml-finance
 source: src/test/daml
-version: 1.2.14.99.20230720.1
+version: 1.2.14.99.20230720.2
```

Comment:
I think we can go back to non pre-release labels, e.g. `1.2.15` given that we will release this version within the next two weeks.

The assembly version doesn't mean much in any case, given that it never becomes a user-facing package